### ISSUE TITLE
2_f demodulation

### ISSUE THREAD

**Issue by GGGGGx** [marceluda/rp_lock-in_pid_h#5]
Hello, do you have 2_f demodulation related settings in this app? At present, only the modulation frequency and demodulation frequency are consistent.

**Comment by marceluda** [comment]
Yes... the F2 signal is for demodulation at 2 f (where f is the base
frequency set on the lock-in panel)

**Comment by GGGGGx** [comment]
thank you.I have find it.
I have a suggestion that when I adjust the modulation signal and Ramp signal,I change the number while the parameters output ,which may be dangerous to the laser.I suggest adding an OK or OUT button,click OK to output paremeters after changing the parameters.
